dataset_listing.tsv

```diff
@@ -82,3 +82,4 @@ mrs_fmrs	Functional MRS data involving a pain stimulus task from 15 subjects	[li
 2d_mb_pcasl	Siemens 2D MultiBand Multi-delay PCASL (m0 and noRF included within timeseries)	[link](https://osf.io/xrkc4/)	[@aptinis](https://github.com/aptinis)	anat, fmap, perf	T1w, asl, aslcontext, epi
 xeeg_hed_score	EEG and iEEG data with annotations of artifacts, seizures and modulators using HED-SCORE 		[@dorahermes](https://github.com/dorahermes)	anat, eeg, ieeg	T1w, channels, coordsystem, eeg, electrodes, events, ieeg
 dwi_deriv	exemplifies the storage of diffusion MRI derivates that may be generated on the Siemens XA platform.			dwi	dwi
+pheno004	dataset containing a mismatch of subjects with MRI data and tabular phenotypic data to be considered a whole dataset of participants for the sake of the participants.tsv file		[@ericearl](https://github.com/ericearl)	anat	T1w, ace, demographics
```

Comment:
@effigies 
seems pybids is indexing the content of phenotype folder and parsing the files trying to get some suffixes: hence why we have ace and demographics in the suffix column here.

is that something we want in terms of behavior for pybids? 

In am not convinced it is but maybe I am missing something.